libraries/AP_UAVCAN/dsdl/com/hex/equipment/flow/20200.Measurement.uavcan

```diff
@@ -0,0 +1,4 @@
+float32 integration_interval    # radians
+float32[2] rate_gyro_integral   # radians
+float32[2] flow_integral        # seconds
+uint8 quality
```

Comment:
How is this interpreted?

Reply:
There's a mistake there I will fix it. Needs to be seconds, radians, radians. I will also add more details.

Reply:
Fixed in https://github.com/ArduPilot/ardupilot/pull/11017/commits/37490b4e60f55834e319499a24f546eaf954f08e

Reply:
What's better 0 or 255?

Reply:
Fixed, thanks